<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextAttrCfger</class>
 <widget class="QGroupBox" name="TextAttrCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>230</height>
   </rect>
  </property>
  <property name="title">
   <string>Text</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="textColor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Text color</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/color_text_64.png</normaloff>image/icon/color_text_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="italic">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_italic_64.png</normaloff>image/icon/text_italic_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="center">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Align center</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_center_64.png</normaloff>image/icon/text_center_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="right">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Align right</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_right_64.png</normaloff>image/icon/text_right_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFontComboBox" name="family">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="point">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Font size</string>
   </property>
   <property name="maximum">
    <number>512</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="bold">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_bold_64.png</normaloff>image/icon/text_bold_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_underline_64.png</normaloff>image/icon/text_underline_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="left">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Align left</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_left_64.png</normaloff>image/icon/text_left_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="textBgColor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Text background color</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/color_text_background_64.png</normaloff>image/icon/color_text_background_64.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="offset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Offset vertical</string>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QPushButton" name="offsetIcon">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Offset vertical</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_offset_64.png</normaloff>image/icon/text_offset_64.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="letterSpaceIcon">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Letter space</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_letter_space_64.png</normaloff>image/icon/text_letter_space_64.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="letterSpace">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Letter space</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="strechHor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Strech horizontal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="strechHorIcon">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Strech horizontal</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/strech_hor_64.png</normaloff>image/icon/strech_hor_64.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="strechVer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Strech vertical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="strechVerIcon">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Strech vertical</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/strech_ver_64.png</normaloff>image/icon/strech_ver_64.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="LineEditor" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pointIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Font size</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_font_size_64.png</normaloff>image/icon/text_font_size_64.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bgColor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Background color</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/color_background_64.png</normaloff>image/icon/color_background_64.png</iconset>
   </property>
  </widget>
  <zorder>italic</zorder>
  <zorder>center</zorder>
  <zorder>right</zorder>
  <zorder>family</zorder>
  <zorder>point</zorder>
  <zorder>bold</zorder>
  <zorder>underLine</zorder>
  <zorder>left</zorder>
  <zorder>textBgColor</zorder>
  <zorder>offset</zorder>
  <zorder>offsetIcon</zorder>
  <zorder>letterSpaceIcon</zorder>
  <zorder>letterSpace</zorder>
  <zorder>strechHor</zorder>
  <zorder>strechHorIcon</zorder>
  <zorder>strechVer</zorder>
  <zorder>strechVerIcon</zorder>
  <zorder>text</zorder>
  <zorder>pointIcon</zorder>
  <zorder>bgColor</zorder>
  <zorder>textColor</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditor</class>
   <extends>QTextEdit</extends>
   <header>LineEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
